recipes/junit2html/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "junit2html" %}
+{% set version = "022" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2160977bec4168babd323da4cf86f5b33ae0220007df0f94abc3f0a57e6d9cdd
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
```

Comment:
It looks like there is a `junit2html` command as well? Can it be tested here?

Reply:
Looks like it can not be tested `ModuleNotFoundError: No module named junit2html`

Reply:
We'll add a test for the junit2html command.

Reply:
Added test for `junit2html` command